document_page_approval/security/document_page_security.xml

```diff
@@ -14,6 +14,7 @@
     <record model="ir.rule" id="rule_change_request_global">
         <field name="name">Change Request Global</field>
         <field name="model_id" ref="model_document_page_history"/>
+        <field name="groups" eval="[(6, 0, [ref('base.group_user')])]"/>
```

Comment:
It makes more sense to use `knowledge.group_document_user` because that's the minimum level of the knowledge app.